CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ include(FeatureSummary)
 if(POLICY CMP0054)
     cmake_policy(SET CMP0054 NEW)
 endif()
```

Comment:
We could remove CMP0054 (introduced in CMake 3.1) since we require CMake 3.12 now...

Reply:
"This policy was introduced in CMake version 3.1. CMake version 3.19.2 warns when the policy is not set and uses OLD behavior. Use the cmake_policy() command to set it to OLD or NEW explicitly." [(source)](https://cmake.org/cmake/help/latest/policy/CMP0054.html)

Reply:
Nevermind, I see that the minimum_required thing overrides that,